crates/wasmtime/src/config.rs

```diff
@@ -2234,6 +2244,11 @@ impl Config {
             bail!("support for GC was disabled at compile time")
         }
 
+        #[cfg(not(feature = "gc"))]
+        if self.enabled_features.contains(WasmFeatures::EXCEPTIONS) {
```

Comment:
This should test `features`-the-local-variable since that represents the calculation of wasm features after enabled/disabled/defaults/etc.

Also mind folding the `#[cfg]` into the boolean expression with `!cfg!(feature = "gc")`? I personally prefer to favor `cfg!` where possible since it avoids dead code warnings and such (not that such motivations are applicable here)

Reply:
Done!